### DISCUSSION TITLE
Open / Close Trades

### DISCUSSION THREAD

**Discussion by cslimani** [Mathieu2301/TradingView-API#62] [General]
Hello,

i would like to open/close  trades, it is possible with your code ?

Cyril

**Comment by Mathieu2301** [comment]
Wow, can you explain more precisely what you want?
This library lets you do just about anything you can do with TradingView. So if you want to backtest a strategy, yes you can!

**Reply by cslimani** [reply]
I would like to trade for real to do a kind of bot.
So for example open a trade with stoploss and takeprofit limts like I do with the GUI.

**Comment by cslimani** [comment]
I check the requests made by TradingView, the trading requests are sent directly toward the broker backend.
In ma case I have to send Http requests to https://cc-tradingview.backend-capital.com

**Reply by Mathieu2301** [reply]
What broker are you using ?

**Comment by cslimani** [comment]
Capital.com

**Reply by mraniki** [reply]
Did you manage to progress on that ?

**Comment by cslimani** [comment]
Comment tu as fait pour trouver comment communiquer avec les serveurs TradingView ?
Tant avec les requêtes HTTP c'est simple mais avec le websocket tu as fait comment ?

**Reply by Mathieu2301** [reply]
L'onglet WebSockets du DevTools

**Comment by sepdoce90** [comment]
@cslimani  Hi, are you trading with Tradingview and Capital.com ? If so, how are you executing your orders. I'm currently testing out Capital.com API with around 3-4 seconds latency from trigger to entry. Please comment if you have a better or faster way, like websocket.
